_data/team/roles/releases_and_deployment.yaml

```
@@ -0,0 +1,18 @@
+role: Releases and deployment
+tasks: 
+  - Monitoring upstream and downstream package changes
+  - Updates and fixes for upstream and downstream packages
+  - Managing future release plans and timing
+  - Carrying out releases
+  - Maintaining continuous deployment infrastructure
+  - Monitoring issues in core library and sub-projects
+  - Carrying out releases for MDAnalysis project packages
+  - Managing released packages on conda-forge and PyPi
```

```suggestion
  - Updates and fixes for upstream and downstream packages
  - Managing future release plans and timing
  - Maintaining continuous deployment infrastructure
  - Carrying out releases for MDAnalysis project packages
  - Managing released packages on conda-forge and PyPi
```

Ok if I remove these, @IAlibay?

> `Monitoring upstream and downstream package changes` and `Monitoring issues in core library and sub-projects` because as you pointed out for the first they don't seem to really fit in a "releases" subrole, and `Carrying out releases` because it now seems redundant with `Carrying out releases for MDAnalysis project packages`

---

Sounds good to me, thanks!